cinder/volume/drivers/lightos/lightos.py

```diff
@@ -834,9 +809,24 @@ def _wait_for_snapshot_state(
 
     def _wait_for_volume_deleted(self, project_name, timeout, vol_uuid):
         """Wait until the volume has been deleted."""
-        return self._wait_for_volume_state(
-            project_name, timeout, states=(
-                'Deleted', 'Deleting', 'UNKNOWN'), vol_uuid=vol_uuid)
+        assert vol_uuid, 'LightOS volume UUID must be specified'
+        states = ('Deleted', 'Deleting', 'UNKNOWN')
```

Comment:
lightOS wont return state Deleted, either one of the active volume states (Aviaalble, updating, creating, migrating, rollback), failed state or deleting
deleted volume will return unkown